lxd/api_metrics.go

```diff
@@ -90,8 +92,8 @@ func metricsGet(d *Daemon, r *http.Request) response.Response {
 		projectNames = []string{projectName}
 	} else {
 		// Get all projects.
-		err := d.cluster.Transaction(func(tx *db.ClusterTx) error {
-			projects, err := tx.GetProjects(db.ProjectFilter{})
+		err := d.db.Transaction(context.Background(), "global", func(ctx context.Context, tx *db.Tx) error {
```

Comment:
Why does this need to use `d.db.Transaction` with "global" rather than using `d.db.Cluster.Transaction` directly?

Reply:
I thought the goal was to phase out the latter's usage and eventually remove it, so I used `d.db.Transaction` wherever it would easily fit.

Reply:
I don't think I'm clear on the goals. The latter option seems to be used in most places? Can you expand on that?

If we are going that way, then we'll need to use a separate type or constant for "cluster" and "local" as relying on strings make searching the code and validating their usage much more difficult.